docs/fsharp/tutorials/getting-started/getting-started-command-line.md

```diff
@@ -18,7 +18,7 @@ This article covers how you can get started with using F# on .NET Core 1.0 with
 
 ## Prerequisites
 
-To begin, you must install the [.NET Core SDK 1.0.0 - Preview 2 (build 003131)](https://dot.net/core).  There is no need to uninstall a previous version of the .NET Core SDK, as it supports side-by-side installations.
+To begin, you must install the [.NET Core SDK 1.0.3 - (build 004769)](https://dot.net/core).  There is no need to uninstall a previous version of the .NET Core SDK, as it supports side-by-side installations.
```

Comment:
is the dot.net site going to be updated to list the RC4 downloads? maybe this should point to the cli repo for now because AFAIK preview2 is the default one until RTM.  also should the build number say build 004769 or later?